platforms/FireTV/RefApp/app/src/main/AndroidManifest.xml

```diff
@@ -19,8 +19,10 @@
         android:hardwareAccelerated="true"
         android:theme="@style/AppTheme">
 
-        <activity
-            android:name=".MainActivity">
+        <activity android:name=".MainActivity"
+            android:alwaysRetainTaskState="true"
```

Comment:
you mentioned in huckleberry that it does not actually resume, does it work here?

Reply:
yes, this is the official way to allow that